go.mod

```diff
@@ -99,6 +99,7 @@ require (
 	gorm.io/driver/postgres v1.4.6
 	gorm.io/driver/sqlserver v1.4.2
 	gorm.io/gorm v1.24.6
+	honnef.co/go/gotraceui v0.2.0
```

Comment:
I don't think this is expected, can you rebase?

Reply:
go mod tidy on main has the same result

Reply:
Apologies, it looks like I missed running `go mod tidy` in #2060. If you'd like, I can make a PR which runs `go mod tidy`?

Reply:
Thanks Nick, feel free to do so. We'll likely not merge this PR any time soon based on a discussion offline